shared/desktop/renderer/index.js

```diff
@@ -93,6 +93,13 @@ function setupApp(store) {
     })
   })
 
+  // After a delay, see if we're connected, and try starting keybase if not
+  setTimeout(() => {
```

Comment:
can we talk through how this works. is the intention to just do this once or on disconnects always?